### PR TITLE
wip: feat(storage-broker): Deploy storage broker

### LINKED ISSUE #1: Deploy storage broker

### PR DESCRIPTION
**Pull request by mimotej**
Resolves: #1

### CODE REVIEW

**Review by schwesig — APPROVED** [review]
/lgtm
/approve

File: storage-broker/base/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+              name: config
+
+      volumes:
+        - emptyDir: {}
+          name: tmpdir
+        - secret:
+            secretName: broker-config
+          name: config-origin
+        - emptyDir: {}
+          name: config
```
Comment (tumido) [inline storage-broker/base/deployment.yaml]: config secret is mounted as volume `config-origin` but above the `config` volume is used and mounted, while `config-origin` remains unused.

https://github.com/AICoE/seraph/pull/23/files#diff-a1804432ce834bdafbcb9f71c5121c29f0949ccdf14373d50ef59dfa3afcc8c3R70-R71

File: storage-broker/base/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - default_map.yaml
```
Comment (tumido) [inline storage-broker/base/kustomization.yaml]: Just a nit: can we rename this to be the same as the resource name please? `storage-broker-map`

File: storage-broker/base/deployment.yaml (hunk 1 above)
Comment (mimotej) [reply storage-broker/base/deployment.yaml]: Thanks :+1: seems that I have forgot it there. Resolved

File: storage-broker/base/kustomization.yaml (hunk 2 above)
Comment (mimotej) [reply storage-broker/base/kustomization.yaml]: Resolved in latest commit, thanks :+1:

**Pull request closed without merging**